utils/connectApp.js

```diff
@@ -91,6 +92,25 @@ const connectApp = async (req, res) => {
       return res.status(200).json({data: shaResult, code: 200});
     }
 
+    else if (api === 'getAppSettings') {
+      if (req.method !== 'GET') {
+        return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+      }
+
+      const selectedParamsArray = req.query.selectedParamsArray
```

Comment:
This is more of a long term design consideration then a comment on this specific code. From a security standpoint as we design apis we should consider why things liked the selected parameters array is optional.  Are there a lot of use cases where the caller is going to need all the data?  If not, then requiring the caller to specify the data they need is both more secure and in the long term more cost effective and performant because it will reduce payload sizes and in turn lower egress costs.

Reply:
@bransteitterbr Great point - agree and updated so `selectedParamsArray` is required. Thanks https://github.com/episphere/connectFaas/pull/612/commits/5d5344818bc205ad193d8f1dfab4ed87baa86cb3